src/cmd/version/version.go

```diff
@@ -0,0 +1,84 @@
+package version
+
+import (
+	"context"
+	cloudclient "github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi"
+	"github.com/otterize/otterize-cli/src/pkg/config"
+	"github.com/otterize/otterize-cli/src/pkg/utils/prints"
+	"github.com/spf13/cobra"
+)
+
+var Version string
+var Commit string
+
+var Cmd = &cobra.Command{
+	Use:          "version",
+	Aliases:      []string{"api-version"},
+	Short:        "Get the Otterize CLI and API versions",
+	Args:         cobra.NoArgs,
+	SilenceUsage: true,
+	RunE: func(_ *cobra.Command, args []string) error {
+		printCLIVersionInformation()
+		printAPIVersionInformation()
+		return nil
+	},
+}
+
+func printAPIVersionInformation() {
+	localAPIVersion, cloudAPIVersion, err := getLocalAndCloudAPIVersions()
+	if err != nil {
+		prints.PrintCliOutput("<failed to load API version>")
+	} else {
+		prints.PrintCliOutput(
+			`Current Cloud API: 
+    version: %s 
+    revision: %s 
+This CLI was built against: 
+    version: %s 
+    revision: %s`,
+			cloudAPIVersion.Version, cloudAPIVersion.Revision,
+			localAPIVersion.Version, localAPIVersion.Revision)
+
+		if cloudAPIVersion != localAPIVersion {
+			prints.PrintCliStderr(`
+Caution: this CLI was built with a different version/revision of the Otterize Cloud API. 
+Some Cloud CLI commands may fail. 
+Upgrade your CLI to the latest build to resolve this issue. 
+For upgrade instructions, see https://docs.otterize.com/getting-started/oss-installation#install-the-otterize-cli
+`)
+		} else {
+			prints.PrintCliOutput(`
+This CLI was built using the latest version & revision of the Otterize Cloud APIs.`)
+		}
+
+	}
+}
+
+func printCLIVersionInformation() {
+	if Version == "v0.0.0" || Version == "" {
+		prints.PrintCliOutput("Version: %s", Commit)
+	} else {
+		prints.PrintCliOutput("Version: %s", Version)
+	}
+}
+
+func getLocalAndCloudAPIVersions() (localAPIVersion cloudclient.APIVersion, cloudAPIVersion cloudclient.APIVersion, err error) {
+	ctxTimeout, cancel := context.WithTimeout(context.Background(), config.DefaultTimeout)
+	defer cancel()
+
+	localAPIVersion, err = cloudclient.GetLocalAPIVersion()
+	if err != nil {
+		return
```

Comment:
Took me a while to figure out that this function does not in fact return nothing: please specify the return values here as well, and don't rely on the default values from the named return values.

It's not possible to look at the `return` and see what is being returned, you have to read the function declaration, and then find references to the return values elsewhere in the code. In other words, the "locality" of the code in a function written in this style is low - can't understand individual bits of code without the rest.